std_msgs/Header        header
geometry_msgs/Vector3  YPR
float32[4]              quat_data

